src/tests/functional/plugin/shared/src/behavior/ov_plugin/caching_tests.cpp

```diff
@@ -184,6 +185,9 @@ void CompileModelCacheTestBase::run() {
         GTEST_COUT << "Plugin doesn't support import and export - skipping test" << std::endl;
         GTEST_SKIP();
     }
+    if (importExportSupported(*core)) {
+        ASSERT_NO_THROW(core->get_property(targetDevice, ov::caching_properties));
+    }
```

Comment:
@ilya-lavrenov is it applicable to add additional checks to the existing tests instead of adding new ones?
And can we merge test changes in a separate PR? Now we're getting failures on HETERO plugin because new property for that haven't introduced

Reply:
we have `SKIP_IF_CURRENT_TEST_IS_DISABLED` which allows to disable the test for specific plugin. Can we try this instead of moving test to a separate PR.

> is it applicable to add additional checks to the existing tests instead of adding new ones?

LGTM